ORK1Kit/ORK1Kit/Onboarding/ORK1LoginStepViewController.m

```diff
@@ -38,11 +38,25 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 #import "ORK1Helpers_Internal.h"
 
 
-@implementation ORK1LoginStepViewController
+@implementation ORK1LoginStepViewController {
+    NSString *_loginButtonTitle;
```

Comment:
I had to refamiliarize myself with the nuances of having an @property declaration in the header, but doing the work manually in the .m file with a custom setter/getter :)

Reply:
Same 😆. Luckily ResearchKit already has several instances of this pattern to confirm I was doing the right thing.